packages/api-core/src/github/fetchers.ts

```diff
@@ -0,0 +1,75 @@
+import { addQueryArgs } from '@wordpress/url';
```

Comment:
Let's rename the directory from `github/` to `hosting-github/` for consistency with the path and the existing structure.

Reply:
Makes sense, done in ec03a01f04a496bacf689ead2b9df4cc5e5c89e6